pkg/module_manager/apis/v1alpha1/doc.go

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright 2023 Flant JSC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+// +k8s:deepcopy-gen=package
+
+//go:generate deepcopy-gen --input-dirs github.com/flant/addon-operator/pkg/module_manager/apis/v1alpha1 -O module_generated.deepcopy --go-header-file ./boilerplate.go.txt -o /tmp
```

Comment:
Is this on CI? If this is a generated spec, we should probably introduce a CI job.